netfilterqueue/_impl.pyx

```diff
@@ -41,14 +48,18 @@ cdef int global_callback(nfq_q_handle *qh, nfgenmsg *nfmsg,
         # so just ignore the packet. The kernel will drop it once we
         # unbind.
         return 1
-    packet = Packet()
+
+    # skipping call to __init__.
```

Comment:
why?

Reply:
the __init__ method is inherently called when instantiating the Packet class and is a pure python method. calling __new__ will return a new instance while reducing unnecessary code execution.

see: https://cython.readthedocs.io/en/latest/src/userguide/special_methods.html?highlight=performance